cypress/support/helper.js

```diff
@@ -140,15 +140,15 @@ export function tableHeaderSize(type) {
 
 export function keyCode(type) {
   return {
-    downarrow: { keyCode: 40, which: 40 },
-    uparrow: { keyCode: 38, which: 38 },
-    leftarrow: { keyCode: 37, which: 37 },
-    rightarrow: { keyCode: 39, which: 39 },
-    Enter: { keyCode: 13, which: 13 },
-    Space: { keyCode: 32, which: 32 },
+    downarrow: { key: "ArrowDown", keyCode: 40, which: 40 },
```

Comment:
@DlgSHi can we remove the `keyCode`s here with it being deprecated or do we still need them?

Reply:
We need to double check :)

Reply:
@samtjo These will be considered and amended for all components altogether in a separate PR.